### PR TITLE
reconfigured add boycott form to a wizard form also including the sea…

### PR DESCRIPTION
**Pull request by jgray7019**
…rch input

will likely lead to a nasty merge conflict with quinn's google places work  😰

### CODE REVIEW

**Review by jesselizdavid — COMMENTED** [review]
hey checked out the branch and having some submit issues so just want to check in with you before we merge :)

File: src/components/BoycottModal.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const mapDispatchToProps = {
 }
 
 const submit = values => {
-    console.log(this.props.updateBoycotts);
+    //console.log(values);
```
Comment (jesselizdavid) [inline src/components/BoycottModal.js]: this is where the issue is happening, where when I click submit, it goes to an error

Comment (jgray7019) [reply src/components/BoycottModal.js]: This is from val and Kritika’s work trying to get it to connect to fire base and I know they’re still working on it?? I was seeing the same error before I made any changes. If you just want to test it’s functionality without it being connected to the backend, if you comment out what’s giving you the error and uncomment console.log(values) then you can see your submission in the the console

Comment (jesselizdavid) [reply src/components/BoycottModal.js]: @jgray7019 yes I see what you mean! I tried to push the commented out change but I think I just pushed to my own local version instead.  Do you want to comment that part out and then merge it?

**Review by jesselizdavid — DISMISSED** [review]

File: src/components/BoycottModal.js (hunk 1 above)
Comment (jgray7019) [reply src/components/BoycottModal.js]: @jesselizdavid ok commented it out but now it won't allow me to merge it since it's my pr

**Review by valmacd — DISMISSED** [review]

**Review by quinn-mccallum — APPROVED** [review]

**Pull request merged**